lib/static/css/main.css

```diff
@@ -1830,13 +1830,17 @@ footer .social {
   padding: 5px 0;
 }
 
-.entry-share {
+.blog-social {
   padding: 36px 0;
   display: block;
 }
 
-@media screen and (max-width: 735px) {
-  .entry-share {
+.blog-social div {
+  padding-bottom: 5px;
+}
+
+@media screen and (max-width: 350px) {
```

Comment:
We can remove this responsive CSS entirely. There are barely any devices that are < 350px these days and we're still loading the content, just not displaying it, so there are not much benefits from hiding it.

Reply:
👍